autoscheduler/scraper/management/commands/fill_terms.py

```diff
@@ -0,0 +1,27 @@
+from django.utils import timezone
+from django.core.management import base
+from scraper.models import Term
+from scraper.management.commands.utils.scraper_utils import get_all_terms
+
+class Command(base.BaseCommand):

```

Comment:
Not a change but I just wanted to comment here as a reminder to run this once this branch gets deployed